data/gui/ghost_replay_info_dialog.stkgui

```diff
@@ -1,16 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
http://www.downloadclipart.net/browse/1612/cartoon-ghost-clipart
![Image](http://www.downloadclipart.net/large/1677-cartoon-ghost-design.png)
Can not be used with any content found on this website for commercial purposes.

Reply:
Right, with everything else I forgot to talk about the picture.

First, let's mention that its unnecessarily large (512px when other icons are 128px). I simply couldn't get GIMP to downscale it with proper antialising, and I gave up. Second, though I edited the picture to (aside from adding the blue button) increase the border's thickness, in some resolutions it doesn't seem entirely right.

Now about the license. This image is found on several websites, as they copy each other. The question then is who is the original source.

I got it from here, where it is listed as CC0 : https://pixabay.com/en/ghost-halloween-ghostly-horror-35852/

Reply:
There is vector_art/open game art icons/ghostrace.svg in media repo 100% original by myself so you can do whatever you like (and i can declare it's public domain right now)

and unknown original source is not acceptable

or couldn't you use a simple blue cross?

Reply:
I first tried to look up for the ghost used in-game, but didn't think about the media repo.

It's possible to use a simple blue cross, sure. I just think it's nicer with a ghost.

I'll sort it out tomorrow - either I can find the original for what I used, or I'll simply do a new version based on yours as then the license question is easy.

Reply:
OK, I think I found the original source : https://openclipart.org/detail/17847/cartoon-ghost

Added in 2008 by a user named lemmling, who also added a few dozens other vector images in around a month, before mostly stopping except two images in 2009 and 2010. Not the sort of profile of someone who's stealing pictures from others.

A japanese illustration site has also tracked down the picture there, see the source button : http://free-illustrations.gatag.net/2013/11/17/190000.html

The ghost's license is indeed CC0.

> We use the Creative Commons Zero 1.0 Public Domain License every time an artist uploads a piece of clipart to Openclipart to make it clear the artist is releasing the creative work for anyone to use for any reason, even commercially. This act of "sharing" is the foundation Openclipart is based upon. Sometimes people want a disclaimer or legal assurance that our clipart is safe to be used. This extra security request often comes from people working for companies of all sizes and, or, publishers. If you need this type of document, please visit, save, download and, or print the following for your needs: Creative Commons Zero 1.0 License.

From : https://openclipart.org/share

Reply:
Then remember to edit License.txt in data/gui